config/pom.xml

```diff
@@ -11,90 +11,30 @@
   <name>Hono Configuration</name>
   <description>Default Hono configuration to be used by other modules.</description>
 
+  <build>
+    <resources>
+      <resource>
+        <directory>demo-certs/certs</directory>
+        <!--<filtering>true</filtering>-->
+      </resource>
+    </resources>
+    <!--<plugins>
```

Comment:
If the only resource we are left with are the certs then I do not see why we would need to filter them.